test/Keywords.test.kt

```diff
@@ -69,4 +69,13 @@
 //       ^ source.kotlin
 //        ^ source.kotlin meta.block.kotlin punctuation.section.group.begin.kotlin
   }
-//^ source.kotlin meta.block.kotlin punctuation.section.group.end.kotlin
\ No newline at end of file
+//^ source.kotlin meta.block.kotlin punctuation.section.group.end.kotlin
+
+suspend fun suspendFunction() {
```

Comment:
Can you move this to a file here https://github.com/nishtahir/language-kotlin/tree/master/test/regression?

The file is usually named after the issue; in this case 45. These are where regression tests go. They usually have the negative matching for scopes.